src/frontend/components/UI/Sidebar/components/SidebarLinks/index.tsx

```diff
@@ -220,13 +220,13 @@ export default observer(function SidebarLinks() {
           </Tooltip>
         </div>
         <div className="sidebarLinkGradientWrapper">
-          <Tooltip {...tooltipProps} label="Wiki">
+          <Tooltip {...tooltipProps} label="Docs">
```

Comment:
i think `Docs` should be a i18n string:
```suggestion
          <Tooltip {...tooltipProps} label={t('sidebar.docs', 'Docs')}>
```

Also, can you also fix the tooltip for Settings above since it also should be an i18n string?

Reply:
Done